health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/AppConfigurationWrapper.js

```
@@ -0,0 +1,217 @@
+    return <Loader />;
+  }
+
+  return (
+    <AppConfigContext.Provider value={{ state, dispatch }}>
+      <AppFieldScreenWrapper />
+      {state?.drawerField && (
+        <SidePanel
+          bgActive
+          className=""
+          closedContents={[]}
+          closedFooter={[<en />]}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)